### ISSUE TITLE
Getting a compilation error on Example 7

### ISSUE THREAD

**Issue by lnbogen** [shomali11/slacker#2]
cannot use "github.com/nlopes/slack".FileUploadParameters literal (type "github.com/nlopes/slack".FileUploadParameters) as type "github.com/shomali11/slacker/vendor/github.com/nlopes/slack".FileUploadParameters in argument to bot.Client.UploadFile

What am I missing?

**Comment by shomali11** [comment]
Could you show me how your imports look like?

Notice that the import should be `github.com/nlopes/slack` for the Slack API as seen in the example below. You might need to do a `go get -u github.com/nlopes/slack` so it is in your `$GOPATH`.

I am leveraging `govendor` from `github.com/kardianos/govendor` to manage my own dependencies and have them bundled under the `vendor` folder. You should not be importing from the vendor folder.

```
package main

import (
	"github.com/nlopes/slack"
	"github.com/shomali11/slacker"
	"log"
)

func main() {
	bot := slacker.NewClient("<YOUR SLACK BOT TOKEN>")

	bot.Command("upload <word>", "Upload a word!", func(request *slacker.Request, response *slacker.Response) {
		word := request.Param("word")
		channel := request.Event.Channel
		bot.Client.UploadFile(slack.FileUploadParameters{Content: word, Channels: []string{channel}})
	})

	err := bot.Listen()
	if err != nil {
		log.Fatal(err)
	}
}
```

**Comment by shomali11** [comment]
I updated the README to include a "Usage" section

# Usage

Using `govendor` [github.com/kardianos/govendor](https://github.com/kardianos/govendor):

```
govendor fetch github.com/shomali11/slacker
```

Using `go get`:

```
go get github.com/shomali11/slacker
go get github.com/nlopes/slack
```

Hopefully this helps. Let me know if it resolves the issue you are facing.

**Comment by lnbogen** [comment]
I did the "go get" from above, of course.

I took the code as is:
cd ~/slacker/examples/
go build example7.go

I think it's related to this: https://github.com/golang/go/issues/12432

To solve it I've added created a slack instance in addition to the slacker one, i.e.
client := slack.New(token) // needed as bot.Client cause conflicts of types.
bot := slacker.NewClient(token)

and then replaced "bot.Client" with client in the example above.

I'm new to Go, so most probably just my lack of understanding on how to explicitly cast the type correctly.
